.github/workflows/coverage.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: "^1.15.4"
+          go-version: "^1.18"
```

Comment:
I think we should get rid of the `^` on these and just pick a specific version so that we don't run into the issue we saw of using a newer version than intended.

Reply:
Wait, let's do this for all of the `go-versions` instead of just this one.